test/integration/assessment/create-assessment.ts

```diff
@@ -0,0 +1,38 @@
+import { UserService } from '@server/service/user'
+import { AssessmentService } from '@server/service/assessment'
+
```

Comment:
do not use dash in file name. this can be called createAssessment. even the editor will recognize the import